crypto/fipsmodule/sha/sha1.c

```diff
@@ -91,7 +91,7 @@ uint8_t *SHA1(const uint8_t *data, size_t len, uint8_t out[SHA_DIGEST_LENGTH]) {
   return out;
 }
 
-#if !defined(SHA1_ASM)
+#if !defined(SHA1_ASM) && !defined(SHA1_ALTIVEC)
```

Comment:
Is the second part `!defined` or `defined`, judging by the beginning of asm/internal.h?

Reply:
I believe this is correct -- `sha1_block_data_order` function is not implemented for ppc64le.

Reply:
I see it declared in crypto/fipsmodule/sha/internal.h at https://github.com/aws/aws-lc/pull/1625/files#diff-390cc5bf910b1a138792200039b10490bcea61b0ad9f5b75484963ac91638611R78

Reply:
Sorry, I meant that it's not implemented in assembly.

Reply:
Sorry, it's just not implemented in `sha1.c` -- it's implemented in `sha1-altivec.c` and declared in `sha/internal.h`